scipy/sparse/linalg/isolve/tests/test_iterative.py

```diff
@@ -16,12 +17,15 @@
 from scipy.sparse import spdiags, csr_matrix, SparseEfficiencyWarning
 
 from scipy.sparse.linalg import LinearOperator, aslinearoperator
-from scipy.sparse.linalg.isolve import cg, cgs, bicg, bicgstab, gmres, qmr, minres, lgmres, gcrotmk
+from scipy.sparse.linalg.isolve import cg, cgs, icgs, bicg, bicgstab, cgnr, cgne, gmres, qmr, minres, lgmres, gcrotmk
 
 # TODO check that method preserve shape and type
 # TODO test both preconditioner methods
 
 
+sysbit = struct.calcsize("P") * 8
```

Comment:
This looks a little unusual. Maybe you want something like `is_64bit = np.dtype(np.intp).itemsize == 8`?

Getting rid of `import struct` would be nice.

Reply:
Yeah. What I need is it, and I'll try to avoid importing external modules, thanks